### ISSUE TITLE
Errors after upgrade from 0.27 to v1 release

### ISSUE THREAD

**Issue by caio-2k** [axios/axios#5203]
### Describe the bug

I'm having a lot of erros after upgrading the axios version from 0.27 to v1 release and actually trying to fix it, but there's one that I can't fix.

I have this code:

![image](https://user-images.githubusercontent.com/59366705/198588401-cc2e48d9-2344-48e3-8f57-3a07e3e46864.png)

![image](https://user-images.githubusercontent.com/59366705/198588264-45fe4874-6077-4419-92f2-b23c8eca883b.png)

And now (after bumping release) I got this error:

![image](https://user-images.githubusercontent.com/59366705/198588905-366d5dab-39ab-4cc4-98cf-13434b1c1c71.png)

Basically it's saying that can't access `data.menu`, can anyone help me?

### Axios Version

0.27.2

### Browser

Chrome

### Additional Library Versions

```bash
Vue 3
```

### Additional context/Screenshots

```bash
httpClient it's my axios create instance:

const httpClient = axios.create({
  //....
});
```

**Comment by ShaktiMandal** [comment]
As per the error, it looks like data is undefined. Make sure response has data props. I think its better to check the network tab first to make sure that API call is made and succeed. In addition, response  has data props.  If it does not, will encounter error.

**Comment by Romick2005** [comment]
Can you please change you code to
```
getMe: async () => {
  try {
    const response = await httpClient.post('/api/auth/me');
    
    console.log("[getMe] response", response);

    return {
      data: response.data
    }
  } catch(error) {
    console.error('[getMe] Error', error);
  }
```

```
try {
  const {data } = await services.users.getMe();

  console.log("[otherMethod] data", data);

  user.setCurrentUser(data);
  
  const path = data.menu;
} catch(error) {
  console.error('[otherMethod] Error', error);
}
```
and provide us output results?

P.S. You should become better at debugging such things ;)

**Comment by jasonsaayman** [comment]
Hi :wave:

Please try the latest pre-release by running the following:

```bash
npm i axios@1.2.0-alpha.1
```

Please provide feedback in either the pinned issue or the discussion thread 🧵
